invenio_rdm_migrator/streams/records/table_generators/records.py

```diff
@@ -12,7 +12,7 @@
 
 from ....load.ids import generate_recid, generate_uuid, pid_pk
 from ....load.models import PersistentIdentifier
-from ....load.postgresql import TableGenerator
+from ....load.postgresql import TableGenerator, _ts
```

Comment:
if we are importing it, we should rename to public method

```suggestion
from ....load.postgresql import TableGenerator, ts
```